activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -1339,7 +1350,7 @@ def arel_column(field)
             lookup_reflection_from_join_dependencies(table)
           end
         else
-          yield field
+          yield Arel::Nodes::SqlLiteral.new(field)
```

Comment:
:thinking: I'm not sure if this is safe. Alternatively it could be possible to support passing string only wrapped by `Arel.sql` already.